modules/ui/deNitro.js

```diff
@@ -0,0 +1,25 @@
+const version = "1.0.0";
+
+let style;
+const css = `.button-38aScr[aria-label="Send a gift"], .channel-2QD9_O[href="/store"], .item-PXvHYJ[aria-controls="Discord Nitro-tab"], .wrapper-3nSjSv, .giftAction-yhDLaW, .upsellFooter-3coAfO, .button-38aScr[aria-label="Open sticker picker"], .navItem-3Wp_oJ[aria-controls="sticker-picker-tab-panel"] {
```

Comment:
Probably better to use alternative selectors rather than aria-label for better compatibility with users using other languages (for the future).